src/js/component.js

```diff
@@ -1181,7 +1190,7 @@ class Component {
     };
 
     // Inherit from this object's prototype
-    subObj.prototype = Lib.obj.create(this.prototype);
+    subObj.prototype = Object.create(this.prototype);
```

Comment:
If this will fail then so will the classes that get generated by babel, because it relies on Object.create too.